src/errorcodes.xml

```diff
@@ -3395,6 +3395,13 @@
         The property is not supported. Consult Starcounter documentation on declaring properties in database classes.
       </message>
     </code>
+    <code hex="0137" severity="Error" name="ScErrNestedTransactionNotSupported">
+      <message>
+        An attempt was made to initialize a new transaction within the scope of an active transaction.
+        Creating such nested transactions is not supported.
```

Comment:
>Creating such nested transactions is not supported.

Only with `Db.Transact`. `Transaction.Create()` is an alternative

Reply:
Shall it be added into the exception message? 😄

Reply:
>Shall it be added into the exception message?

I don't know, but you're suggesting alternatives (`Consider reusing the already active transaction or running a new transaction outside the already active transaction's scope`) and the list of alternatives is not full without it.

Reply:
Let's just have it in the docs. Will be available when searching by `ScErrNestedTransactionNotSupported`.